x/pool-incentives/keeper/grpc_query.go

```diff
@@ -88,3 +90,27 @@ func (k Keeper) IncentivizedPools(ctx context.Context, _ *types.QueryIncentivize
 		IncentivizedPools: incentivizedPools,
 	}, nil
 }
+
+func (k Keeper) ExternalIncentiveGauges(ctx context.Context, req *types.QueryExternalIncentiveGaugesRequest) (*types.QueryExternalIncentiveGaugesResponse, error) {
```

Comment:
Can you add godoc comments for this function, and some comments inside the function to explain it?

Reply:
Done in https://github.com/osmosis-labs/osmosis/pull/572/commits/cbabd871a117231d1e3db4a0257d46d555c098de.